console.go

```diff
@@ -2,26 +2,64 @@ package otto
 
 import (
 	"fmt"
-	"os"
-	"strings"
+	"io"
 )
 
-func formatForConsole(argumentList []Value) string {
-	output := []string{}
-	for _, argument := range argumentList {
-		output = append(output, fmt.Sprintf("%v", argument))
-	}
-	return strings.Join(output, " ")
+type Console interface {
+	Log(v ...interface{})
+	Trace(v ...interface{})
+	Debug(v ...interface{})
+	Info(v ...interface{})
+	Warn(v ...interface{})
+	Error(v ...interface{})
 }
 
-func builtinConsole_log(call FunctionCall) Value {
-	fmt.Fprintln(os.Stdout, formatForConsole(call.ArgumentList))
-	return Value{}
+type console struct {
+	out io.Writer
 }
 
-func builtinConsole_error(call FunctionCall) Value {
-	fmt.Fprintln(os.Stdout, formatForConsole(call.ArgumentList))
-	return Value{}
+func (c *console) log(v ...interface{}) {
+	fmt.Fprintln(c.out, v...)
```

Comment:
Can't be configured so just use directly
```suggestion
	fmt.Fprintln(os.Stdout, v...)
```